tenable/io/v3/was/configurations/api.py

```diff
@@ -0,0 +1,411 @@
+'''
+Configurations
+==============
+
+The following methods allow for interaction into the Tenable.io
+:devportal:`configurations <was-v2-configurations>` API.
+
+Methods available on ``tio.v3.was.configurations``:
+
+.. rst-class:: hide-signature
+.. autoclass:: ConfigurationsAPI
+    :members:
+'''
+from typing import Dict, List, Optional
+from uuid import UUID
+
+from tenable.io.v3.base.endpoints.explore import ExploreBaseEndpoint
+from tenable.io.v3.was.configurations.schema import ConfigurationSchema
+
+
+class ConfigurationsAPI(ExploreBaseEndpoint):
```

Comment:
Can we make a generic base class for both WAS and VM here?

Reply:
@tomeara, We are using `ExploreBaseEndpoint` as the generic base class for all the APIs, i.e., whether they are under `vm` or `was` or on the outer level. The configurations API is present only in WAS, hence, we have kept it under `was` only.
Please let us know if we are missing anything over here or if there are any further changes required in the class design/ structure.